Testreglar/3.1.1/App/app-3.1.1a.json

```diff
@@ -0,0 +1,106 @@
+{
+	"namn": "App-3.1.1a Hovedspråket på appsiden er programmatisk bestemt 2023",
+	"id": "app-3.1.1a",
+	"testlabId": 480,
+	"versjon": "1.0",
+	"type": "App",
+	"spraak": "nb",
+	"kravTilSamsvar": "<ul>\r\n<li>Appsiden har en gyldig språkkode og</li>\r\n<li>Språkkoden som er angitt stemmer med hovedspråket på den aktuelle appsiden</li>\r\n</ul>",
+	"side": "2.1",
+	"element": "Side",
+	"kolonner": [
+		{
+			"title": "2.2"
+		},
+		{
+			"title": "2.3"
+		},
+		{
+			"title": "3.2"
+		}
+	],
+	"steg": [
+		{
+			"stegnr": "2.1",
+			"spm": "Hvilken appside tester du?",
+			"ht": "<p>Beskriv appsiden med få stikkord, eller velg i listen.</p>",
+			"type": "tekst",
+			"label": "Appside:",
+			"datalist": "Sideutvalg",
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.2"
+				}
+			}
+		},
+		{
+			"stegnr": "2.2",
+			"spm": "Er det mulig å sveipe til innhold på appsiden?",
```

Comment:
Bør det spesifiseres tidlig at språket i innstillinger på Talkback og VoiceOver må være riktig satt og at språket må være lagt til? Hvis jeg på VO ikke har lagt skrudd på fastslå språk kommer den alltid til å snakke med "gjeldende stemme (Norsk for meg)".

Reply:
@janBeniaminKwiek Har du forslag til hvordan en slik hjelpetekst/presisering kan formuleres?

Reply:
Fastlå språk skal være av. 
Skjermgjenkjenning under VoiceOver-gjenkjenning skal være av.
Sjekk at relevant språk er lastet ned.